### PR TITLE
Make users sortable by draftspace characters added in students tab

### LINKED ISSUE #1612: In Students tab, users should be sortable by draft space characters added

### PR DESCRIPTION
**Pull request by psinghal20**
Fixes #1612.

### CODE REVIEW

File: app/assets/javascripts/components/students/students_handler.jsx [hunk 1]
```diff
@@ -45,6 +45,7 @@ const StudentsHandler = createReactClass({
               {lastNameSorting}
               <option value="character_sum_ms">{I18n.t('users.characters_added_mainspace')}</option>
               <option value="character_sum_us">{I18n.t('users.characters_added_userspace')}</option>
+              <option value="character_sum_ds">{I18n.t('users.characters_added_draftspace')}</option>
```
Comment (ragesoss) [inline app/assets/javascripts/components/students/students_handler.jsx]: This is the wrong name for that data; it's `character_sum_draft`.

File: app/assets/javascripts/stores/user_store.js [hunk 2]
```diff
@@ -5,7 +5,8 @@ const UserStore = new StockStore({
   sortAsc: true,
   descKeys: {
     character_sum_ms: true,
-    character_sum_us: true
+    character_sum_us: true,
+    character_sum_ds: true
```
Comment (ragesoss) [inline app/assets/javascripts/stores/user_store.js]: Likewise: `character_sum_draft`.

Comment (psinghal20) [reply app/assets/javascripts/stores/user_store.js]: Should I do the same changes in locales files also? @ragesoss

Comment (ragesoss) [reply app/assets/javascripts/stores/user_store.js]: No. These come from the `CoursesUsers` model, but they aren't user-facing labels. Changing them in these two places will fix the sorting.

The labels are actually pretty bad, and changing them to something clearer would be helpful, that's independent of this fix.

Comment (psinghal20) [reply app/assets/javascripts/stores/user_store.js]: I can try to change the labels if you feel that would be better.

**Pull request merged**